compiler/luci/export/src/CircleOperationExporter.cpp

```diff
@@ -890,7 +890,7 @@ void OperationExporter::visit(luci::CircleSpaceToDepth *node)
 {
   export_simple(node, circle::BuiltinOperator_SPACE_TO_DEPTH,
                 circle::BuiltinOptions_SpaceToDepthOptions,
-                CreateSpaceToDepthOptions(builder).Union());
+                CreateSpaceToDepthOptions(builder, node->block_size()).Union());
```

Comment:
maybe unintended change?

Reply:
Sorry for make changing in two module in on PR!

I will make a PR for Export first and change luci value test later.